connection.js

```diff
@@ -162,13 +162,6 @@ class Connection {
             if (gamedata.phase === "finished" && !(gamedata.id in this.connected_games))
```

Comment:
I wonder if it would be clearer if all handling for finished games was in one place:

```
// Handle finished games.
if (gamedata.phase === "finished") {
    if (!gamedata.id in this.connected_games) {
        return;
    } else {
        setTimeout(...);
    }
}

// Handle in-progress games.
...
```

Reply:
no idea at all :smile: 
but feedback always appreciated :100:

Reply:
@windo I made the change, see what you think before I merge? I also made the logging based on DEBUG. Thought about removing it completely, but I guess it isn't bad to log that.